frontend/components/forms/validators/valid_email/valid_email.ts

```diff
@@ -1,4 +1,7 @@
-const EMAIL_REGEX = /\S+@\S+\.\S+/;
+// see https://stackoverflow.com/a/201378
+
+// eslint-disable-next-line no-control-regex
+const EMAIL_REGEX = /(?:[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9]))\.){3}(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])/;
```

Comment:
I'm not too concerned about this because it is running client-side, so if a user is tracking to hack the input with many `\\�` they'll just crash their own browser. That's a heck of a regex, though! Were there some specific edge cases we were trying to catch?

Reply:
lol you'll have to ask @jacobshandling. I assume it was just pulled from that StackOverflow link